<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsuariosPage</class>
 <widget class="QWidget" name="UsuariosPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="logoLabel">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>180</y>
     <width>104</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>731</width>
     <height>376</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="OficinaLbl">
      <property name="text">
       <string>Oficina</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPlainTextEdit" name="OficinaInp"/>
    </item>
    <item row="3" column="1">
     <widget class="QPlainTextEdit" name="AccesosInp"/>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="NombreInp"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="NombreLbl">
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPlainTextEdit" name="ApellidoInp"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="ApellidoLbl">
      <property name="text">
       <string>Apellido</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Commit">
      <property name="text">
       <string>Agregar Usuario</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="EdadLbl">
      <property name="text">
       <string>Edad</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPlainTextEdit" name="EdadInp"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="AccesosLbl">
      <property name="text">
       <string>Accesos</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>500</y>
     <width>731</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QPlainTextEdit" name="EliminarInp"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="EdadLbl_2">
      <property name="text">
       <string>Uid</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="Commit_2">
      <property name="text">
       <string>Eliminar Usuario</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
